src/setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = ambianic-fall-detection
+version = 1.0.0
+author = Ivelin Ivanov
```

Comment:
@bhavikapanara add your name to the authors list

Reply:
done